src/adapters/MorphoMarketV1Adapter.sol

```diff
@@ -35,6 +35,8 @@ contract MorphoMarketV1Adapter is IMorphoMarketV1Adapter {
     address public skimRecipient;
     /// @dev `shares` are the recorded shares created by allocate and burned by deallocate.
     mapping(Id => uint256) public shares;
+    Id[] public marketArray;
+    mapping(Id => uint256) public marketMapping;
```

Comment:
this is to be more efficient upon removal right? why do we care about this? it shouldn't happen much no?